### DISCUSSION TITLE
Round window corners

### DISCUSSION THREAD

**Discussion by aolko** [FancyWM/fancywm#211] [Q&A]
Is it possible to implement rounded window corners on wm level?

**Comment by veselink1** [comment]
It is not possible to do so within FancyWM, and I don't believe there is a Windows setting to do so unfortunately.

**Reply by aolko** [reply]
hmmm (win11 only) https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/apply-rounded-corners
